yarn.lock

```diff
@@ -1599,6 +1599,24 @@
     rollup-pluginutils "^2.3.3"
     slash "^2.0.0"
 
+"@ovh-ux/manager-pci@^0.6.0-alpha.0":
```

Comment:
`yarn install` from `638e7cf552013681804993326826d993f320a275` causes this following change.

Reply:
It should have been merged with `638e7cf552013681804993326826d993f320a275` then 😞

Reply:
Rebased on `9723d478fdb549390a75074d8047b6dac668d1a5`.